app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::CommentsController < ApplicationController
+  def index
+    if request.headers['X-AUTH-TOKEN']
+      @user = User.find_by_api_token(request.headers['X-AUTH-TOKEN'])
+      @comments = Comment.where(user_id: @user.id)
+      respond_to do |format|
+        format.json { render json: @comments, status: :ok }
+      end
+    else
+      respond_to do |format|
+        format.json { render json: 'please sign in or add the token to create comment'.to_json, status: :ok }
+      end
+    end
+  end
```

Comment:
- [ ] You should have an API endpoint that returns all comments for a post - **GET :** `/users/:user_id/posts/:id/comments`. Here you are missing a reference to the `post` where to fetch the list of comments.

<img width="1048" alt="Screenshot 2022-12-09 at 2 07 39 PM" src="https://user-images.githubusercontent.com/40503704/206709356-0fc3cfc9-a6cd-4651-ae00-42f55f368bea.png">

Reply:
**I have done based on review**

![image](https://user-images.githubusercontent.com/65109017/206744082-23d39369-5f23-49a5-893a-709fe4a5c5ff.png)